### DISCUSSION TITLE
Δήλωση παρουσίας στην τακτική συνάντηση

### DISCUSSION THREAD

**Discussion by epidrome** [courses-ionio/iv#12] [Announcements]
Στα παρακάτω σχόλια μπορείτε να κάνετε δήλωση παρουσίας για τις διαθέσιμες συναντήσεις που είναι κάθε 

Δευτέρα, Τετάρτη, και Παρασκευή στις 8μμ στην Αίθουσα Τηλεδιάσκεψης στον Γαληνό.

Οι συναντήσεις γίνονται αποκλειστικά με ραντεβού μετά την έγκαιρη (δύο ή τρεις ημέρες πριν) ανάρτηση του σχολίου σας παρακάτω, έτσι ώστε να υπάρχει χρόνος να οργανώσουμε έγκαιρα τις ομάδες συνάντησης κάθε εβδομάδα.

Ο στόχος είναι να έχουμε τακτικές παραγωγικές συναντήσεις με μικρές ομάδες.

Για αυτόν τον σκοπό θα πρέπει να έρχεστε στην συνάντηση αφού πρώτα έχετε ολοκληρώσει τις πρόσφατες δραστηριότητες του προγράμματος του μαθήματος έτσι ώστε να κάνετε επίδειξη της προόδου σας. Ανάλογα με την προσέλευση ενδέχεται να αλλάξουμε την μέρα και ώρα της συνάντησης.

Παρακαλούμε να απαντάτε στα παρακάτω σχόλια μόνο με την δήλωση παρουσίας για το αντίστοιχο σχόλιο-εβδομάδα για όποια άλλη απορία ανοίξτε νέα συζήτηση

**Comment by epidrome** [comment]
Τετάρτη 5 Οκτ. 8μμ Γαληνός

| ΑΜ |
| - |
| 2003ΧΧΧ|
| |

Παρασκευή 7 Οκτ. 8μμ Γαληνός

| ΑΜ |
| - |
| 2003ΧΧΧ|
| |

**Reply by epidrome** [reply]
Δηλώστε παρακάτω με μια απάντηση σε αυτό το σχόλιο τον ΑΜ για την παρουσία σας για την μέρα-ώρα που σας βολεύει

**Comment by epidrome** [comment]
Δευτέρα 10 Οκτ. 7μμ Γαληνός

| ΑΜ |
| - |
| 2020064 |
| |

Παρασκευή 14 Οκτ. 8μμ Γαληνός

| ΑΜ |
| - |
| 2003ΧΧΧ|
| |

**Reply by epidrome** [reply]
Δηλώστε παρακάτω με μια απάντηση σε αυτό το σχόλιο τον ΑΜ για την παρουσία σας για την μέρα-ώρα που σας βολεύει

**Reply by heremolaos** [reply]
Δευτέρα 10 Οκτ. 8μμ Γαληνός

|   **ΑΜ**  |
|  ----------- |
| 2020064 |

**Reply by epidrome** [reply]
@GeorgiosEleftheriadis 
σε βολεύει αύριο Δευτέρα 7μμ; την Παρασκευή μπορώ και στις 8μμ

**Reply by heremolaos** [reply]
@epidrome Ναι βεβαίως, 7μμ είναι καλά. Σας ευχαριστώ.

**Comment by epidrome** [comment]
Καθώς το ενδιαφέρον για το μάθημα είναι πολύ μικρό μπορείτε να κλείνετε ατομικές συναντήσεις για την παρουσίαση της προόδου σας μόνο στις ώρες του μαθήματος στον σύνδεσμο: http://calendly.com/choko/progress
